py-polars/pyproject.toml

```diff
@@ -26,11 +26,17 @@ profile = "black"
 disallow_untyped_defs = true
 warn_unused_ignores = true
 show_error_codes = true
-files = ["polars", "tests"]
+files = ["polars"]
```

Comment:
We want the tests to be type checked. This has uncovered many typing inconsistencies and shows to improve type hints over time.

Reply:
Oh, apologies, that wasn't supposed to be in this PR! Will change this.